.github/actions/deploy/action.yml

```diff
@@ -0,0 +1,132 @@
+name: deploy
+description: deploys application
+
+inputs:
+  actions-api-access-token:
+    required: true
+  azure_credentials:
+    required: true
+  environment:
+    description: Environment to deploy to
+    required: true
+  pr:
+    description: PR number for the review app
+    required: false
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  slack-webhook:
+    required: true
+  terraform-state-access-key:
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Start ${{ inputs.environment }} Deployment
+      uses: bobheadxi/deployments@v0.6.2
+      id: deployment
+      with:
+        step: start
+        token: ${{ inputs.actions-api-access-token }}
+        env: ${{ inputs.environment }}
+        ref: ${{ inputs.sha }}
+
+    - name: Set Environment variable
+      shell: bash
+      run: |
+        if [ -n "${{ inputs.pr }}" ]; then
+          DEPLOY_ENV=review
+          echo "DEPLOY_ENV=review" >> $GITHUB_ENV
+          echo "DEPLOY_URL=https://find-pr-${{ inputs.pr }}.london.cloudapps.digital" >> $GITHUB_ENV
+        else
+          echo "DEPLOY_ENV=$DEPLOY_ENV" >> $GITHUB_ENV
+        fi;
+
+        tf_vars_file=terraform/workspace_variables/${DEPLOY_ENV}.tfvars.json
+        echo "key_vault_name=$(jq -r '.key_vault_name' ${tf_vars_file})" >> $GITHUB_ENV
+        echo "key_vault_app_secret_name=$(jq -r '.key_vault_app_secret_name' ${tf_vars_file})" >> $GITHUB_ENV
+        echo "key_vault_infra_secret_name=$(jq -r '.key_vault_infra_secret_name' ${tf_vars_file})" >> $GITHUB_ENV
+        echo "cf_space=$(jq -r '.cf_space' ${tf_vars_file})" >> $GITHUB_ENV
```

Comment:
Is this used anywhere?

Reply:
Doesn't look like `cf_space` is used anywhere, remove it